course_discovery/apps/api/v1/views/course_runs.py

```diff
@@ -13,6 +13,8 @@
 from course_discovery.apps.course_metadata.constants import COURSE_RUN_ID_REGEX
 from course_discovery.apps.course_metadata.models import CourseRun
 
+from course_discovery.apps.api.edly_utils import get_edly_sub_organization
```

Comment:
@iam-mhaseeb where I can find the implementation of `get_edly_sub_organization`?

Reply:
@ziafazal It was missed in the previous commit. For now, it's dummy function it's actual implementation would be done after cookie setting flow when the user logs in.